db/migrate/20230313104452_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :password
+      t.string :phone
+      t.boolean :role
```

Comment:
Do you think we should accept null value for these attributes?
You can look for db validations e.g. null: false, default: 'something'.
Thank you!

Reply:
Hello apu, editing migration is okay, but when i apply that, it bring all the upcoming migration in this schema file in this branch. should i keep those in this branch?

Reply:
you can drop the DB, create a new DB, then write your single migration file, which will eventually have a schema. As it's almost a blank DB, dropping the DB will not be an issue here. 
Please reach out if you have any confusion. Thanks!

Reply:
Resolved apu.. Thanks for the guide 😇